docs/getting-started/cases-req.asciidoc

```diff
@@ -8,4 +8,36 @@ For more information, see
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges].
 
 NOTE: To send cases to external systems, you need the
-https://www.elastic.co/subscriptions[appropriate license].
\ No newline at end of file
+https://www.elastic.co/subscriptions[appropriate license].
+
+You can create roles and define feature privileges at different levels to manage feature access in {kib}. {kib} privileges grant access to features within a specified {kib} space and full or partial access can be given.
+
+When setting up feature access to cases, make sure to configure the following:
+
+- Base privileges for the {es-sec} solution
+- Sub-feature privileges for the Cases feature
```

Comment:
Should we indicate that this step is optional? They really only need to configure the sub-feature privileges for the Cases feature if they want it to be different than the base privilege. That might be implied though.

Reply:
Good point, this is an important distinction to make. I'll also include a line at the start of the topic describing what the default Case priv settings are.